### PR TITLE
Include 'mobile_browser' param when requesting JITMs. This will allow mobile targeted messages

### PR DESCRIPTION
**Pull request by jtreanor**
#### Changes proposed in this Pull Request:

We would like to introduce Just in Time Messages (JITMs) that encourage users to download the WordPress mobile apps. These JITMs should only be delivered on mobile devices. Currently, the API has no awareness of the user agent or any other way of knowing what browser is used.

This PR adds a param `mobile_browser`  to the query string when requesting JITMs from the API. It indicates if the request is coming from a mobile browser.

@withinboredom do you think this is an appropriate way to pass this context to the API? We could pass the full UA, but that may be excessive.

#### Testing instructions:

Follow these steps on a desktop browser and a mobile device (such as the iOS Simulator):

* Go to wp-admin
* Observe API requests from Jetpack to `/jitm` in the the API.
* Mobile requests will have `?mobile_browser=1` and desktop requests will have `?mobile_browser=0`.

#### Proposed changelog entry for your changes:

I don't think a changelog entry is needed for this.

### CODE REVIEW

File: class.jetpack-jitm.php
Comment (withinboredom) [inline class.jetpack-jitm.php]: Yep, looks good. This also needs to be whitelisted in the api endpoint in wpcom.

**Review by withinboredom — DISMISSED** [review]

File: class.jetpack-jitm.php
Comment (jeherve) [inline class.jetpack-jitm.php]: Could you add spaces inside the brackets, as per WordPress' coding standards?

**Review by jeherve — CHANGES_REQUESTED** [review]
Minor changes to make our linter happy. Other than that, it should be good to go.

File: class.jetpack-jitm.php
Comment (jtreanor) [reply class.jetpack-jitm.php]: This is now updated.

**Review by jeherve — APPROVED** [review]
This looks good to me. 👍 	Could you get someone from your team to review this PR as well? Once we have one more approval, feel free to merge!

Thank you!

**Review by withinboredom — APPROVED** [review]

**Comment by oskosk** [comment]
Related: D19945-code

**Pull request merged**
